src/objects/objects.js

```diff
@@ -9,12 +9,12 @@ const AnimationManager = require("../animation/AnimationManager.js");
 const CSS2D = require("./CSS2DRenderer.js");
 const SkeletonUtils = require("../SkeletonUtils.js");
 
-function Objects(){
-
+function Objects(map) {
+	Objects.prototype.map = map;
+	Objects.prototype.animationManager = new AnimationManager(this.map)
```

Comment:
This is essentially global on the Objects class but unfortunately some functions (Object3D) use Objects.prototype directly instead of making an Objects instance.